ethereum/eth/src/test/java/tech/pegasys/pantheon/ethereum/eth/transactions/PendingTransactionsTest.java

```diff
@@ -43,8 +47,18 @@
   private static final KeyPair KEYS2 = KeyPair.generate();
 
   private final MetricsSystem metricsSystem = new NoOpMetricsSystem();
+
+  final Vertx vertx = Vertx.vertx();
+  final TimerUtil timerUtil = new VertxTimerUtil(vertx);
```

Comment:
We should be using a `MockTimerUtil` here so that we can test things in a. single thread and avoid creating a Vertx instance. Also note that every created Vertx must be closed or threads leak and cause problems for other tests.